packages/@headlessui-react/src/components/transition/transition.test.tsx

```diff
@@ -1050,8 +1050,8 @@ describe('Events', () => {
         'child-2-2: beforeEnter',
 
         'child-1: afterEnter',
-        'child-2-2: afterEnter',
         'child-2-1: afterEnter',
+        'child-2-2: afterEnter',
```

Comment:
Do you know why the order of these changed?

Reply:
I could revert the change to the test, and apply the diff above. But it feels incorrect since that would rely on us accessing the `finished` property, instead of relying on the creation of the `CSSTransition` object.

Reply:
yeah that's fine — was just curious and hoping it wasn't a side effect of some "problem"

Reply:
sweet 👍